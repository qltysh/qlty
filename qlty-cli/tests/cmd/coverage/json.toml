args = [
  "coverage",
  "publish",
  "--dry-run",
  "--print",
  "--json",
  "--override-commit-sha",
  "2ca1bc45a94e37c8dbae6fd9e19fc069ba64bd67",
  "--transform-add-prefix",
  "rails/",
  "--tag",
  "rails",
  "--summary",
  "lcov.info"
]
bin.name = "qlty"

[env.add]
GITHUB_ACTIONS = "true"
GITHUB_HEAD_REF = ""
GITHUB_REF = ""
GITHUB_REF_NAME = "feature-branch-1"
GITHUB_REF_TYPE = "branch"
GITHUB_RUN_ATTEMPT = "2"
GITHUB_RUN_ID = "42"
GITHUB_SHA = "77948d72a8b5ea21bb335e8e674bad99413da7a2"
QLTY_COVERAGE_TOKEN = "qltcp_12345678abcdefgh"
QLTY_CI_UPLOADER_TOOL = "qltysh/qlty-action-coverage"
QLTY_CI_UPLOADER_TOOL_VERSION = "v1.0.0"
