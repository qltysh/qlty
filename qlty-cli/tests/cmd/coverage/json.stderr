qlty [..] [..]-[..] ([..] debug [..])
https://qlty.sh/d/coverage

 SETTINGS 

    cwd: [CWD]
    dry-run: true
    tag: rails
    override-commit-sha: 2ca1bc45a94e37c8dbae6fd9e19fc069ba64bd67
    transform-add-prefix: rails/

 METADATA 

    CI: GitHub
    Commit: 2ca1bc45a94e37c8dbae6fd9e19fc069ba64bd67
    Branch: feature-branch-1
    Build ID: 42:2

 COVERAGE FILES: 1 

    Coverage File  Format  Size
    lcov.info      lcov    903 B

 COVERAGE DATA 

    1 unique code file paths
    1 paths are missing on disk (100.0%)

    Missing code files:

      formatter.js

    TIP: Consider using add-prefix or strip-prefix to fix paths
    https://qlty.sh/d/coverage-path-fixing


    Covered Lines:      24
    Uncovered Lines:     2

    Line Coverage        92.31%

 EXPORTING... 

    Exported: tmp/qlty-coverage/coverage.zip

